mlflow/version.py

```diff
@@ -1,3 +1,4 @@
 # Copyright 2018 Databricks, Inc.
 
 VERSION = '0.2.1' #  NOQA
+NEXT_VERSION = '0.2.2' #  NOQA
```

Comment:
I think it would be better to change VERSION to the NEXT_VERSION in master automatically when doing release. Or maybe to switch it to a dev version.